### PR TITLE
fix: add Amr Monier

### PR DESCRIPTION
**Pull request by AmrMonier**
added new json file **AmrMonier.json** to the ```/public/data``` 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/799"></a>

### CODE REVIEW

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
On line 4. Please can you remove the comma at the end of your bio @AmrMonier 
Thank you

**Comment by CodingSpecies** [comment]
Sorry at @AmrMonier didn’t see your second commit where you fixed your syntax

**Comment by AmrMonier** [comment]
@CodingSpecies  no problem

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
Actually, @AmrMonier I realised that you fixed every other syntax but this one:
image
the comma is in the string. After, that, I will approve. Thank you.

**Comment by AmrMonier** [comment]
@CodingSpecies i have just fixed it

File: public/data/AmrMonier.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "Amr Monier",
+  "type": "personal",
+  "bio": "Backend Developer, JS enthusiast, and a learning JUNKY😁😁",
```
Comment (Panquesito7) [inline public/data/AmrMonier.json:4]:
```suggestion
  "bio": "Backend Developer, JS enthusiast, and a learning JUNKY 😁",
```

File: public/data/AmrMonier.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      "url": "https://github.com/AmrMonier",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/amr_monier",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (Panquesito7) [inline public/data/AmrMonier.json:18]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/AmrMonier.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      "description": "started the compulsory military service for a year where worked as a PL/SQL developer. lucky me i have been a technical solider."
+    },
+    {
+      "title": "Finished my Military Service",
+      "date": "December 2021",
+      "color": "green",
+      "description": "the  first of December 2021 the day i finished my military service"
+    },
+    {
+      "title": "Looking for an opportunity as Backend web devloper",
```
Comment (Panquesito7) [inline public/data/AmrMonier.json:57]:
```suggestion
      "title": "Looking for opportunities as a Backend Web Developer",
```

File: public/data/AmrMonier.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      "date": "present",
+      "color": "green",
+      "description": "currentl i'm buuilding side projects and looking for a job as Backend Developer"
```
Comment (Panquesito7) [inline public/data/AmrMonier.json:60]:
```suggestion
      "description": "Currently, I'm building side-projects and looking for a job as a Backend Developer"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**